frontend/webapp/containers/main/actions/choose-action-modal/index.tsx

```diff
@@ -28,50 +28,64 @@ const SubTitle = styled(Text)`
   line-height: 150%;
 `;
 
+const Center = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+
 interface AddActionModalProps {
   isModalOpen: boolean;
   handleCloseModal: () => void;
 }
 
-interface ModalActionComponentProps {
-  onNext: () => void;
-}
-
-const ModalActionComponent: React.FC<ModalActionComponentProps> = React.memo(({ onNext }) => {
-  const buttons = [
-    {
-      label: 'DONE',
-      onClick: onNext,
-      variant: 'primary' as const,
-    },
-  ];
-
-  return <NavigationButtons buttons={buttons} />;
-});
-
 export const AddActionModal: React.FC<AddActionModalProps> = ({ isModalOpen, handleCloseModal }) => {
-  const submitRef = useRef<(() => void) | null>(null);
+  const { formData, handleFormChange, resetFormData, validateForm } = useActionFormData();
+  const { createAction, loading, done } = useCreateAction();
```

Comment:
and use it in the isComplete function
```suggestion
const { createAction, loading } = useCreateAction(handleClose); 
```